package.json

```diff
@@ -1,15 +1,31 @@
 {
   "name": "min-reviewers-action",
-  "main": "index.js",
+  "main": "build/index.js",
   "scripts": {
-    "build": "standard --fix index.js && ncc build index.js -o dist --source-map --license licenses.txt"
+    "build": "tsc",
+    "format": "prettier --write 'src/*.ts'",
+    "format-check": "prettier --check 'src/*.ts'",
+    "package": "ncc build --license licenses.txt",
+    "lint": "eslint {tests,src}/**/*.ts",
+    "test": "jest",
+    "ci": "npm run test && npm run lint && npm run format-check && npm run build && npm run package",
+    "release": "npm run test && npm run lint && npm run format && npm run build && npm run package"
   },
   "dependencies": {
     "@actions/core": "^1.10.0",
     "@actions/github": "^5.1.1"
   },
   "devDependencies": {
+    "@octokit/rest": "^19.0.5",
+    "@types/jest": "^29.2.5",
+    "@types/node": "^18.11.0",
+    "@typescript-eslint/eslint-plugin": "^5.48.0",
+    "@typescript-eslint/parser": "^5.48.0",
     "@vercel/ncc": "^0.36.0",
```

Comment:
Maybe I'm wrong, but this package is not used.

Reply:
`package.json` -> `script` -> `package`